easy-ui-react/src/Notification/Notification.stories.tsx

```diff
@@ -0,0 +1,136 @@
+import { InlineStoryDecorator } from "../utilities/storybook";
+
+type Story = StoryObj;
+
+const meta: Meta<typeof Notification> = {
+  title: "Components/Notification",
+  argTypes: {},
+};
+
+export default meta;
+
+export const Notifications: Story = {
+  render: () => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const { notification } = useNotification();
+
+    return (
+      <>
+        <Button
+          onPress={() => notification.showSuccessToast({ message: "Success" })}
+        >
+          Success Toast
+        </Button>
+        <Button
+          onPress={() => notification.showSuccessAlert({ message: "Success" })}
+        >
+          Success Alert
+        </Button>
+        <Button
+          onPress={() =>
+            notification.showPromotionalToast({ message: "Promotional" })
+          }
+        >
+          Promotional Toast
+        </Button>
+        <Button
+          onPress={() =>
+            notification.showPromotionalAlert({ message: "Promotional" })
+          }
+        >
+          Promotional Alert
+        </Button>
+        <Button
+          onPress={() => notification.showNeutralToast({ message: "Neutral" })}
+        >
+          Neutral Toast
+        </Button>
+        <Button
+          onPress={() => notification.showNeutralAlert({ message: "Neutral" })}
+        >
+          Neutral Alert
+        </Button>
+        <Button
+          onPress={() => notification.showWarningToast({ message: "Warning" })}
+        >
+          Warning Toast
+        </Button>
+        <Button
+          onPress={() => notification.showWarningAlert({ message: "Warning" })}
+        >
+          Warning Alert
+        </Button>
+        <Button
+          onPress={() => notification.showErrorToast({ message: "Error" })}
+        >
+          Error Toast
+        </Button>
+        <Button
+          onPress={() => notification.showErrorAlert({ message: "Error" })}
+        >
+          Error Alert
+        </Button>
+        <Button
+          onPress={() =>
+            notification.showSuccessToast({
+              message: "Success",
+              hasIcon: false,
+            })
+          }
+        >
+          Without icon
+        </Button>
+        <Button
+          onPress={() =>
+            notification.showSuccessAlert({
+              message: "Success",
+              onDismiss: () => alert("alert has been dismissed"),
+            })
+          }
+        >
+          Dismiss callback
+        </Button>
+      </>
+    );
+  },
+
+  decorators: [InlineStoryDecorator],
+};
+
+export const CustomPlacement: Story = {
+  render: () => {
+    return (
+      <EasyUIProvider
+        colorScheme="system"
+        notificationPlacementOffset={{
+          top: "50px",
+          left: "0px",
+          right: "0px",
+          bottom: "0px",
+        }}
+      >
+        <SimulatedNotificationTrigger message="Hello World" />
+      </EasyUIProvider>
+    );
+  },
+};
```

Comment:
so technically, if someone issues a notification with the `Notification` story above and then comes to this story to issue another notification while the other is still visible, a warning is logged to the console: 

`Page contains more than one landmark with the 'region' role and 'Notifications' label. If two or more landmarks on a page share the same role, they must have unique labels`

but im more willing to accept this scenario since this story highlights the custom placement aspect

Reply:
i wasn't able to replicate the console warning but does it do this because it's wrapped in multiple providers, one at the parent and one here?

Reply:
it comes from `react-aria` `useLandmark` [hook](https://github.com/adobe/react-spectrum/blob/main/packages/%40react-aria/landmark/src/useLandmark.ts#L217) which is used behind the scenes by `useToastRegion`.